cli/commands/backend/migrate/migrate.go

```diff
@@ -0,0 +1,37 @@
+// Package migrate provides the ability to bootstrap remote state backend.
+package migrate
+
+import (
+	"context"
+
+	"github.com/gruntwork-io/terragrunt/config"
+	"github.com/gruntwork-io/terragrunt/internal/errors"
+	"github.com/gruntwork-io/terragrunt/internal/remotestate/backend"
+	"github.com/gruntwork-io/terragrunt/options"
+)
+
+func Run(ctx context.Context, srcPath, dstPath string, opts *options.TerragruntOptions) error {
+	remoteState, err := config.ParseRemoteState(ctx, opts)
```

Comment:
This is parsing remote state configurations for the current directory? If so, why? Shouldn't this always be parsing state in a particular directory?

Reply:
> This is parsing remote state configurations for the current directory?
No, it works the same as for other commands, mainly takes into account `working-dir`, and so on.